### PR TITLE
feat(documentation): adds deprecation message to badge

### PR DESCRIPTION
**Pull request by b1aserlu**

### CODE REVIEW

File: packages/documentation/src/stories/components/badge/badge.docs.mdx [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 import * as badgeStories from './badge.stories';
 import StylesPackageImport from '../../../shared/styles-package-import.mdx';
 
+
 <Meta of={badgeStories} />
 
+<PostAlert type="warning">
+  <p slot="heading">Badge is deprecated</p>
+  <p>
+    The current badge will be deprecated in favour of a new chip component that looks similar to it. There will also be a new badge component
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/components/badge/badge.docs.mdx:12]:
```suggestion
    The current badge is deprecated and will be replaced with the new chip component that will look and work similar like the current badge component. There will also be a new badge component, but with a different behavior.
```

File: packages/documentation/src/stories/components/badge/badge.docs.mdx [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+    that looks similar to the current nested badge.
+    Check out the new components in figma:
+    [Chip](https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?type=design&node-id=11194-50777&mode=design&t=SeiMf2Vt3dmUnpMT-0)
+    /
+    [Badge](https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?type=design&node-id=11038-51303&mode=design&t=SeiMf2Vt3dmUnpMT-0)
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/components/badge/badge.docs.mdx:17]:
```suggestion
```

File: packages/documentation/src/stories/components/badge/badge.docs.mdx [hunk 3]
```diff
@@ -1,9 +1,22 @@
 import { Canvas, Controls, Meta } from '@storybook/blocks';
+import {  PostAlert } from '@swisspost/design-system-components-react';
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/components/badge/badge.docs.mdx:2]:
```suggestion
import { PostAlert } from '@swisspost/design-system-components-react';
```

File: packages/documentation/src/stories/components/badge/badge.docs.mdx [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
 import * as badgeStories from './badge.stories';
 import StylesPackageImport from '../../../shared/styles-package-import.mdx';
 
+
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/components/badge/badge.docs.mdx:6]:
```suggestion
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]
@b1aserlu There seems to be something wrong with your autoformat functionality!

File: .vscode/settings.json [hunk 5]
```diff
@@ -1,9 +1,5 @@
 {
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "[mdx]": {
-    "editor.defaultFormatter": null,
-    "editor.formatOnSave": false
-  },
```
Comment (imagoiq) [inline .vscode/settings.json:7]: This should not be removed (or at least first discussed and in another PR dedicated to it).

**Review by imagoiq — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 5 above)
Comment (b1aserlu) [reply .vscode/settings.json]: I do not know why we cannot use prettier to format mdx files as well. For me sometimes the formatting does not work in mdx and I think its due to this lines. VSCode tells me that it isnt sure what formatter to use with mdx files and then it acts all confuesed. I reverted the change for now though.

File: packages/documentation/src/stories/components/badge/badge.docs.mdx [hunk 6]
```diff
@@ -1,9 +1,21 @@
 import { Canvas, Controls, Meta } from '@storybook/blocks';
+import { PostAlert } from '@swisspost/design-system-components-react';
 import * as badgeStories from './badge.stories';
 import StylesPackageImport from '../../../shared/styles-package-import.mdx';
 
 <Meta of={badgeStories} />
 
+<PostAlert type="warning">
```
Comment (imagoiq) [inline packages/documentation/src/stories/components/badge/badge.docs.mdx:8]: Nitpick: Just for design coherency, can you please put the alert after the lead like https://next.design-system.post.ch/?path=/docs/components-alert--docs

File: packages/documentation/src/stories/components/badge/badge.docs.mdx [hunk 7]
```diff
@@ -1,9 +1,21 @@
...
+  <p slot="heading">Badge is deprecated</p>
+  <p>
+    The current badge is deprecated and will be replaced with the new chip component that will look
+    and work similar like the current badge component. There will also be a new badge component, but
```
Comment (imagoiq) [inline packages/documentation/src/stories/components/badge/badge.docs.mdx:12]:
```suggestion
    The current badge is deprecated and will be replaced with the chip component which will behave similarly as the current badge component. There will also be a new badge component, but
```
Might be a bit better?

**Review by oliverschuerch — APPROVED** [review]

**Review by imagoiq — APPROVED** [review]

**Pull request merged**